src/js/api/config.js

```diff
@@ -10,7 +10,6 @@ define([
         //androidhls: true,
         autostart: false,
         controls: true,
-        cookies: true,
```

Comment:
Do we still want this config option to control whether settings are stored? It's a feature.